TTHAnalysis/python/analyzers/ntupleTypes.py

```diff
@@ -1,6 +1,6 @@
 #!/bin/env python
 from math import *
-from PhysicsTools.Heppy.analyzers.core.autovars import NTupleObjectType  
+from PhysicsTools.Heppy.analyzers.core.autovars import NTupleObjectType
```

Comment:
You shouldn't do any changes on this file... Remember that we want to make changes only to the Stop4Body specific parts.  Revert all changes on this file. It is mostly from spaces that your editor automatically removed from the end of lines.